pom.xml

```diff
@@ -103,6 +114,11 @@
       <artifactId>maven-invoker</artifactId>
       <version>2.2</version>
     </dependency>
+    <dependency>
+      <groupId>org.projectlombok</groupId>
+      <artifactId>lombok</artifactId>
+      <version>1.18.12</version>
+    </dependency>
```

Comment:
Please do not use Lombok - write getters/setters directly in the code - it's just for one class

Reply:
Fixed